lpr.WebAPI/Controllers/OrganisationController.cs

```diff
@@ -37,6 +37,21 @@ public class OrganisationController : ControllerBase {
       return StatusCode(200, org);
     }
 
+    [HttpGet("GetPackagesPaginated/{page}/{amount}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<IActionResult> GetOrganisationsPaginated(int page,
+                                                               int amount) {
+      try {
```

Comment:
This is not now a problem, but in the future, the try-catch inside a controller needs to be removed for a global error resolver